apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -27,6 +32,21 @@ const styles = {
     fontSize: 13,
     fontWeight: 'bold'
   },
+  button: {
+    backgroundColor: color.purple,
+    borderWidth: 0,
+    color: color.white,
+    fontSize: 'larger',
+    paddingTop: 12.5,
+    paddingBottom: 12.5,
```

Comment:
Unfortunately fractional pixels are necessary to give the button the correct height:

12px:
<img width="749" alt="publish-button-12px" src="https://user-images.githubusercontent.com/8001765/28796826-e2d67df8-75f3-11e7-8711-0bf3953c672f.png">

12.5px:
<img width="764" alt="publish-button-12 5px" src="https://user-images.githubusercontent.com/8001765/28796832-ea080006-75f3-11e7-8d5b-0c78d7c6de6b.png">

13px:
<img width="769" alt="publish-button-13px" src="https://user-images.githubusercontent.com/8001765/28796837-f0faccf4-75f3-11e7-9353-b18f8ba22e4e.png">

I've confirmed that the button height looks right @ 12.5px in Firefox, Safari and IE11 too.